experimental/sdk/webpubsub-socketio-extension/src/SIO/index.ts

```diff
@@ -42,6 +83,40 @@ export async function useAzureSocketIOChain(
   return this;
 }
 
+function getDefaultNegotiateHandler(): (
+  req: IncomingMessage,
+  res: ServerResponse,
+  getGenerateClientTokenOptions: (req: IncomingMessage) => Promise<GenerateClientTokenOptions>,
+  serviceClient: WebPubSubServiceClient
+) => Promise<void> {
+  return async (
+    req: IncomingMessage,
+    res: ServerResponse,
+    getGenerateClientTokenOptions: (req: IncomingMessage) => Promise<GenerateClientTokenOptions>,
+    serviceClient: WebPubSubServiceClient
+  ): Promise<void> => {
+    let statusCode = 500;
+    let message = "Internal Server Error";
+    try {
+      const generateClientTokenOptions = await getGenerateClientTokenOptions(req);
+
+      // Example: https://<web-pubsub-endpoint>?access_token=ABC.EFG.HIJ
+      const endpointWithToken = (await serviceClient.getClientAccessToken(generateClientTokenOptions)).url
+        .replace("ws://", "http://")
+        .replace("wss://", "https://")
+        .replace(`/client/hubs/${serviceClient.hubName}`, "");
+      statusCode = 200;
+      message = endpointWithToken;
+    } catch (e) {
+      statusCode = 500;
+      message = e.message;
+    } finally {
+      res.writeHead(statusCode, { "Content-Type": "text/plain" });
```

Comment:
use application/json json output incase we want to add more properties in the future